src/main/java/com/gsheet/student/controller/StudentController.java

```diff
@@ -0,0 +1,63 @@
+package com.gsheet.student.controller;
+
+import com.gsheet.student.dto.StudentDto;
+import com.gsheet.student.entity.Student;
+import com.gsheet.student.service.StudentService;
+import com.gsheet.student.transformer.StudentToStudentDtoTransformer;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/students")
+@RequiredArgsConstructor
+public class StudentController {
+
+    @Autowired
+    private StudentService studentService;
+
+    @Autowired
+    private StudentToStudentDtoTransformer studentToStudentDtoTransformer;
+
+    @SneakyThrows
+    @GetMapping
+    public List<StudentDto> index() {
+        return studentService.getStudents()
+            .stream()
+            .map(studentToStudentDtoTransformer::transform)
+            .toList();
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<StudentDto> get(@PathVariable int id) {
+        Student student = studentService.getStudent(id);
+
+        return ResponseEntity.ok(studentToStudentDtoTransformer.transform(student));
```

Comment:
If you don't have some specific logic to handle on the response you should simply return Student object, Spring will automatically wrap it and return OK status, as you have done on `index` method